config/db_schemas/forum.schema.js

```diff
@@ -48,9 +49,9 @@ const forumSchema = new Schema({
     attachments: [{
         type: String
     }],
-    comments: [
-        //TODO: Change to use 'commentSchema' once it has been created
-    ]
+    comments: [{
+        // Comment
```

Comment:
```suggestion
        // TODO: Change to use 'commentSchema' once it has been created
```
I think it would be better to leave the TODO so others know it is not implemented

Reply:
I made changes so that it now actually stores the comments.